src/transfer.ts

```diff
@@ -74,3 +74,53 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+interface TransferAssetParams extends SendTransactionParams {
```

Comment:
This interface and the method should both have full jsdoc comments for the documentation and intellisense